packages/modules/web-viewer-react/package.json

```
@@ -73,8 +73,8 @@
     "@itwin/core-frontend": "^4.0.0",
     "@itwin/core-react": "^4.0.0",
     "@itwin/presentation-common": "^4.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^17.0.2 || ^18.2.0",
```

sorry lets set peers for react & react-dom to 

```suggestion
    "react": "^17.0.2 || ^18.0.0",
```

---

changed